tests/kernel/threads/thread_stack/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_ZTEST=y
+CONFIG_INIT_STACKS=y
+CONFIG_THREAD_STACK_INFO=y
+CONFIG_APPLICATION_DEFINED_SYSCALL=y
+CONFIG_TEST_USERSPACE=y
```

Comment:
Do you intentionally leave HW_STACK_PROTECTION off the list here (might be already default but it makes sense to explicitly add it here, no?)

Reply:
this test does not exercise HW_STACK_PROTECTION. It does implictly get enabled if ztest is active, but there is no specific scenario for it in this test suite. We test that stuff in other suites (tests/kernel/fatal IIRC)